src/index.js

```diff
@@ -1,44 +1,116 @@
-import _ from 'lodash';
 import './style.css';
 import More from './more.png';
+import Delete from './delete.png';
 
-const cardData = [
-  {
-    description: 'Wash the dishes',
-    completed: false,
-    index: '3',
-  },
-  {
-    description: 'Complete the project',
-    completed: false,
-    index: '2',
-  },
-  {
-    description: 'Submit for code review',
-    completed: false,
-    index: '1',
-  },
-];
+function getItems() {
+  const value = localStorage.getItem('tasks') ? JSON.parse(localStorage.getItem('tasks')) : [];
+  return value;
+}
+
+function setItems(items) {
+  const itemsJson = JSON.stringify(items);
+  localStorage.setItem('tasks', itemsJson);
+}
 
 const postContainer = document.querySelector('.tasklist');
+const addButton = document.querySelector('.newtask-return');
+const items = getItems();
+
+function updateItem(index, key, value) {
```

Comment:
- [ ] Kindly 🙏🏻 follow the best practices for the `JavaScript` by using the `arrow` functions or ES6 style JavaScript code instead of `ES5` functions because ES6 syntax makes your code more readable, manageable and clean. Even the functionalities provided in the `ES6`, results are more predictable and reliable. Please fix the things as I mentioned.